### PR TITLE
TELCODOCS-1319: URGENT: Removing OSC content from OCP docs starting from 4.13…

### PR DESCRIPTION
**Pull request by MirzWeiss**
Starting from OCP release 4.13, Sandboxed containers documentation is moving to its own doc set at access.redhat.com. 

This PR is to remove all existing content from main and 4.13 (and 4.14). Older versions of the docs will remain in the OCP guide. **This PR isn't actually an XXL!**

We have left Sandboxed containers in the topic map and edited the single page to link to the new content for OSC users. The product shipped live this morning.

Version(s): Main, 4.13, 4.14

Issue: [TELCODOCS-1319](https://issues.redhat.com/browse/TELCODOCS-1319)

[Link to docs preview](https://60931--docspreview.netlify.app/openshift-enterprise/latest/sandboxed_containers/sandboxed-containers-moved.html)

QE review:
- [x] QE has approved this change.

### CODE REVIEW

**Comment by tbuskey** [comment]
QE
/lgtm

File: sandboxed_containers/sandboxed-containers-moved.adoc [hunk 1]
```diff
@@ -0,0 +1,11 @@
+:_content-type: ASSEMBLY
+[id="sandboxed-containers-moved"]
+= {sandboxed-containers-first} documentation has been moved
+include::_attributes/common-attributes.adoc[]
+:context: sandboxed-containers-moved
+
+Starting from {product-title} 4.13, {sandboxed-containers-first} documentation has moved to a new location.
```
Comment (opayne1) [inline sandboxed_containers/sandboxed-containers-moved.adoc:7]: Just a suggestion for wording. 
```suggestion
Documentation for {sandboxed-containers-first} has moved to a new location. Refer to the following:
```
or
```suggestion
Starting from {product-title} 4.13, {sandboxed-containers-first} documentation has moved to a new location. Refer to the following:
```

**Review by opayne1 — COMMENTED** [review]
removal looks good to me. Just some suggestions on the wording in the assembly file. 

Looping in @kalexand-rh for an FYI

File: sandboxed_containers/sandboxed-containers-moved.adoc (hunk 1 above)
Comment (MirzWeiss) [reply sandboxed_containers/sandboxed-containers-moved.adoc:7]: Updated.

**Comment by tradej** [comment]
We're having an issue where the [updates](https://github.com/openshift/openshift-docs/compare/main...MirzWeiss:openshift-docs:TELCODOCS-1319) that Miriam did to this PR aren't showing. If the PR is not refreshed by tomorrow, can we ask that a replacement PR with the same changes that Miriam will create be merged right away, please?

**Comment by tradej** [comment]
Never mind, the PR was updated

**Comment by abhatt-rh** [comment]
> @MirzWeiss the CPs failed for 4.13 and 4.14. Please do a manual cherrypick and I can merge those. Thanks!

@opayne1 I happened to notice this comment, which made me think about this guideline in the merge review checklist:

> If the cherry-pick bot fails to open a PR automatically for the latest version branch, there’s probably a larger issue with the PR. Attempt to investigate it, and ask Andrea if you cannot locate the issue.

cc: @bergerhoffer

**Comment by bergerhoffer** [comment]
Thanks a lot for tagging me @abhatt-rh! We do always want to investigate any 4.14 failures.

It looks like there were some differences between main and 4.14 (which should normally be in sync). That's usually a problem, but luckily in this case, it doesn't really matter anymore since we are just removing all the files and replacing with a note that it has moved. So in this case, I think it's fine to continue with the manual cherry pick to 4.14. After these PRs merge, main and 4.14 should be in sync again.

Thanks again!

**Comment by bergerhoffer** [comment]
@MirzWeiss @tradej I see that the 4.13 CP has been done, but 4.14 has not yet. Can one of you please take care of the cherry pick to 4.14?

**Comment by bergerhoffer** [comment]
@tradej thanks for the update. However, main and 4.14 have to stay in sync, so no matter what, you'll need to cherry pick this change to 4.14.

Then if you decide you want to remove the section altogether, you'd submit that as a new PR against main and CP to 4.14.

**Comment by tradej** [comment]
Of course, will do.

**Comment by MirzWeiss** [comment]
@opayne1 @bergerhoffer cherry-pick to 4.14 is ready to merge.

**Comment by bergerhoffer** [comment]
@opayne1 do you mind taking care of reviewing/merging the cherry pick since you've done these others?

**Pull request merged**